pkg/api/client/clientset/versioned/typed/operators/v1/operators_client.go

```diff
@@ -54,12 +56,28 @@ func (c *OperatorsV1Client) OperatorGroups(namespace string) OperatorGroupInterf
 }
 
 // NewForConfig creates a new OperatorsV1Client for the given config.
+// NewForConfig is equivalent to NewForConfigAndClient(c, httpClient),
+// where httpClient was generated with rest.HTTPClientFor(c).
 func NewForConfig(c *rest.Config) (*OperatorsV1Client, error) {
```

Comment:
Should we be keeping these NewForConfigs around? Are they still going to be used?